gtars-core/src/models/region_set.rs

```diff
@@ -531,6 +532,56 @@ impl RegionSet {
         }
         total_count
     }
+
+    ///
+    /// Calculate statistics
+    ///
+    pub fn calculate_statistics(&self) -> HashMap<String, ChromosomeStats> {
```

Comment:
This returns chrom stats per chrom, right? Should we maybe make the function + doc string reflect that? Maybe:
```rust
fn calculate_chromosome_statistics
```